src/config/address_format.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2011-2019 libbitcoin developers (see AUTHORS)
+ *
+ * This file is part of libbitcoin.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <bitcoin/explorer/config/address_format.hpp>
+
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <boost/program_options.hpp>
+#include <bitcoin/system.hpp>
+#include <bitcoin/explorer/define.hpp>
+
+namespace libbitcoin {
+namespace explorer {
+namespace config {
+
+using namespace bc::system;
+using namespace bc::system::wallet;
+using namespace po;
+
+static const auto address_format_p2wpkh = "p2wpkh";
+static const auto address_format_p2wsh = "p2wsh";
```

Comment:
Acronyms :/

Reply:
I ended up mapping p2wpkh to `witness_pubkey_hash` and p2wsh to `witness_script_hash`.  Hopefully more clear now.